inspire/utils/author_tokenizer.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
```

Comment:
Almost perfect: can you move the content of this file under `inspire/modules/authors/utils.py`. IMHO it should belongs there. Sorry if that was not clear in my previous comments.